src/app/content/components/Sidebar.tsx

```diff
@@ -31,8 +32,15 @@ const SidebarPlaceholder = styled.div<{isOpen: boolean}>`
   `}
 `;
 
-const SidebarControl = styled(({isOpen, ...props}: React.HTMLProps<HTMLButtonElement> & {isOpen: boolean}) =>
-  <button {...props}>
+type StyledSidebarControlProps = React.HTMLProps<HTMLButtonElement> & {
+  isOpen: boolean,
+  intl: IntlShape & InjectedIntl
+};
+
+const StyledSidebarControl = styled(({isOpen, intl, ...props}: StyledSidebarControlProps) =>
+  <button {...props}
+    aria-label={intl.formatMessage({id: isOpen ? 'i18n:toc:toggle:opened' : 'i18n:toc:toggle:closed'})}
```

Comment:
is there any reason to use formatMessage here instead of the FormattedMessage component?

Reply:
ooh, that might work. I'll try. I used the example from https://github.com/yahoo/react-intl/wiki#formatting-data but consistency with `FormattedMessage` would be nicer